src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -154,6 +154,11 @@ internal static class IDEDiagnosticIds
 
         public const string RemoveUnnecessaryDiscardDesignationDiagnosticId = "IDE0110";
 
+        // 2000 range for experimental formatting enforcement
+        public const string WrapEmbeddedStatementDiagnosticId = "IDE2000";
+        public const string ConsecutiveBracePlacementDiagnosticId = "IDE2001";
+        public const string MultipleBlankLinesDiagnosticId = "IDE2002";
+
```

Comment:
If these are expected to be non-experimental after some time, won't we have to use other IDs, which is breaking?

Reply:
1.  Anything experimental doesn't have to worry about breaking. That's the point of being experimental. 
2. I don't see why we'd need to change the ids...

Reply:
> 2. I don't see why we'd need to change the ids...

My point on this was that if you'll keep the 2000 range for experimental, they ids will need to change to something else. But that should be okay based on:

> Anything experimental doesn't have to worry about breaking. That's the point of being experimental.

Reply:
Yes. It should be ok. What I meant though was simply to put these rules in the 2000 range apart from everything else.  I can envision lots of these rules, and if like them separated from the rest.